.github/workflows/lint-test.yaml

```diff
@@ -3,32 +3,6 @@ name: Lint and Test Charts
 on: pull_request
 
 jobs:
-  lint:
```

Comment:
I can't comment on the first line since it is out of the scope of the diff, but... can you please change the name of this action? It is not linting anymore